sections/02_intro.md

```diff
@@ -1,3 +1,25 @@
 ## Introduction
 
-Intro goes here.
+### Potential writing prompt
+
+One potential future that we could imagine is a world in which data, once
+gathered, is rapidly integrated into our understanding of living systems
+(STUDY). What we learn is rapidly put to use. Our health-related activities are
+constantly monitored (e.g. by wearables) and all of our interactions with health
+care systems are extensively tracked. These sources of information are combined
+to help to guide our health care and maintenance. (CATEGORIZE) We'd be able to
+compare our state and trajectory to (anonymized) others, and identify means to
+improve our health. (TREAT) These means might contain drug combinations selected
+based on personalized predictions.
+
+### If this happens, is deep learning required for any of it? Are we any closer
+### because of the advent of deep learning?
+
+* "Categorize" and "treat" sound a bit like PMI goals. Another way to think
```

Comment:
Bullets here (and elsewhere) seem to be induced by atom's convenient "reflow selection" functionality. I tracked it down here and in the other part, and expanded PMI. Thanks!